data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -172,11 +172,12 @@ public String getMigratedOpenApi() {
   }
 
   private void addStandardApiHandlers(
+      final Spec spec,
       final DataProvider dataProvider,
       final EventChannels eventChannels,
       final AsyncRunner asyncRunner,
       final BeaconRestApiConfig configuration) {
-    addBeaconHandlers(dataProvider);
+    addBeaconHandlers(spec, dataProvider);
```

Comment:
dataProvider usually first if we want to be consistent...

Reply:
Yeah it's a bit of a toss up because spec is usually first... :)

Reply:
Just handlers haven't previously had spec.

Reply:
ok well its probably inconsistent in there i guess... pretty sure other handlers had spec already but could be wrong

Reply:
I don't mind switching it around.  I'd be surprised if handlers had spec before because we usually kept them away from all the internal details with the old reflection based API.  But 🤷‍♂️